proxy/stickiness.go

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/openvpn-forwarder" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package proxy
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+
+	log "github.com/cihub/seelog"
```

Comment:
After migrating to seelog timestamps are with mixed style & non human readable.
Maybe we let live with Go's default logger until we uncouple from `goproxy` code

```
2019/10/03 23:34:58 [006] INFO: Got request / api.ipify.org GET http://api.ipify.org/?format=text
1570134898080236000 [Warn] Failed to load IP to UserID mapping: not found
1570134898080269000 [Info] Falling back to IP-stickiness
2019/10/03 23:35:05 [006] INFO: Copying response to client 200 OK [200]
2019/10/03 23:35:05 [006] INFO: Copied 14 bytes to client error=<nil>
```

Reply:
Done.